dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/BlockingFunction/BlockingFunction.mo

```diff
@@ -60,7 +62,7 @@ equation
   when time - TimerStartBlock > 0 then
     blocked = true;
     TimerMaintainBlock = time;
-  elsewhen time - TimerStartBlock < 0 and time > pre(TimerMaintainBlock) + TBlock and time > pre(TimerPrepareDeblock) + TDeblockU then
+  elsewhen time - TimerStartBlock < 0 and time > pre(TimerMaintainBlock) + TBlock and time > pre(TimerPrepareDeblock) + TDeblockU + TBlock then
```

Comment:
Why this change ?

Reply:
to match with Eurostag deblocking function !